sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -486,4 +489,22 @@ object QueryExecution {
     val preparationRules = preparations(session, Option(InsertAdaptiveSparkPlan(context)), true)
     prepareForExecution(preparationRules, sparkPlan.clone())
   }
+
+  /**
+   * Catches asserts and illegal state exceptions, and converts them to internal errors.
+   */
+  private[sql] def withInternalError[T](msg: String)(block: => T): T = {
+    try {
+      block
+    } catch {
+      case e: SparkThrowable => throw e
```

Comment:
do we need this line?

Reply:
We needed when we had `SparkIllegalStateException` which extends `IllegalStateException`. After we removed it, we don't need to handle `SparkThrowable` somehow especially. I will remove this line. Thank you.